src/accelerate/state.py

```diff
@@ -500,7 +501,17 @@ def _split_values(inputs, start_index, end_index):
                     inputs[key] = _split_values(inputs[key], start_index, end_index)
                 return inputs
             else:
-                return inputs
+                if is_datasets_available():
+                    from datasets import Dataset
+
+                if isinstance(inputs, Dataset):
```

Comment:
probably this should also be indented to be within the scope of if-condition in line 504

Reply:
right! thanks